src/sumlog.jl

```diff
@@ -1,42 +1,94 @@
 """
-$(SIGNATURES)
+    sumlog(X::AbstractArray{T}; dims)
 
-Compute `sum(log.(X))` with a single `log` evaluation.
+Compute `sum(log.(X))` with a single `log` evaluation,
+provided `float(T) <: AbstractFloat`.
```

Comment:
This sounds as if the function requires `float(T) <: AbstractFloat` which is not the case.

```suggestion
Compute `sum(log, x; dims=dims)`.

If `float(eltype(x)) <: AbstractFloat` the computation is performed with a single `log` evaluation.
```